helm/observability-bundle/values.yaml

```diff
@@ -45,6 +45,30 @@ userConfig:
             enabled: false
 
 apps:
+  alloy:
+    appName: alloy
+    chartName: alloy
+    catalog: giantswarm
+    enabled: false
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/alloy-app
+    version: 0.3.0
+    # a list of extraConfigs for the App,
+    # It can be secret or configmap
+    # https://github.com/giantswarm/rfc/tree/main/multi-layer-app-config#example
+    extraConfigs:
+      - kind: configMap
+        name: "{{ $.Values.clusterID }}-observability-bundle-psp-override"
+        namespace: "{{ $.Release.Namespace }}"
+        priority: 150
+      - kind: secret
+        name: "{{ $.Values.clusterID }}-logging-secret"
```

Comment:
This is expected to be the same configmap than promtail?

Reply:
Yes this is expected